doc/source/ray-core/actors.rst

```diff
@@ -89,6 +89,28 @@ that specific worker and can access and mutate the state of that worker.
           // as the argument.
           auto counter = ray::Actor(CreateCounter).Remote();
 
+
+.. tip::
+    
+    From Ray 2.0 onwards, you could also use :ref:`Ray's State API <state-api-overview-ref>``  to query the actors information, or go to the :ref:`Dashboard <observability-getting-started>`` Actor page to see more details. (These features all require installing ray with `pip install "ray[default]"`). 
```

Comment:
Let's just say "you can see if actors are created using ray list actors or the dashboard (link)" -> and have a short note saying it is only available for 2.5. Something similar to pg doc


<img width="954" alt="Screen Shot 2023-05-25 at 3 10 48 PM" src="https://github.com/ray-project/ray/assets/18510752/7d6c95d9-7501-43fe-99e0-c16ef4ba4077">